### ISSUE TITLE
Do not parse CSS for variables

### ISSUE THREAD

**Issue by varya** [SC5/sc5-styleguide#533]
Here https://github.com/SC5/sc5-styleguide/blob/dev/lib/modules/variable-parser.js#L70 and here https://github.com/SC5/sc5-styleguide/blob/dev/lib/modules/variable-parser.js#L105 `visitors.css = visitors.sass;` has been added.

I think it is better not to parse CSS for variables at all rather than parse it as SASS.

CSS will have variables, this is the draft http://www.w3.org/TR/css-variables/ But their syntax is going to be different, not what we have now in SASS.

It is better to prevent parsing at all. For example, with returning `[]` in the beginning of these functions:

```
if (syntax === 'css') { // No variables in CSS
    return [];
}
```

Or, even better, not to call these methods if syntax is `css`.

**Comment by cyberixae** [comment]
Makes sense. It could be considered a bug but I am marking it as enhancement since I think the only risk is that someone will accidentally use the css file suffix for their sass files.

**Comment by d2s** [comment]
Does this problem also apply if file is named as `visitors.css.sass` or `visitors.css.scss` (for example)?

**Comment by cyberixae** [comment]
Actually, support for standard CSS variables would be nice.
